### ISSUE TITLE
Increase traffic gate from 50% to 100% from Virtual Agent Chatbot Team (target 6/13 or later)

### ISSUE THREAD

**Issue by ianpsant** [department-of-veterans-affairs/va.gov-cms#9332]
## Description
Previously in #8766, on the Contact Us page (https://www.va.gov/contact-us/) we increased the traffic gate for unauthenticated traffic being able to see the link to the Virtual Agent Chatbot page (https://www.va.gov/contact-us/virtual-agent/) from 25% to 50%. We now want to increase this to 100%. 

Previous PR, for code reference: https://github.com/department-of-veterans-affairs/vets-website/pull/21002/files

For now we want to keep this gate to give us the option of being able to control the percentage who can see the link. If this functionality is no longer needed, we'll file a separate ticket to remove this code from the Contact Us page.

### Timing
The earliest we'd want this released is Monday afternoon June 13th, but it is dependent on tasks on our end, and obvoiusly the Public Websites team. Please check with us to confirm the release date, or let us know when you can take this on.

## Acceptance Criteria
- [ ] GIVEN user on Contact Us page THEN they will see the Info Alert containing the link to the Virtual Agent Chatbot 100% of the time
- [ ] Requires design review

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [X] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by jilladams** [comment]
Yep, that works fine. Thanks!

**Comment by ianpsant** [comment]
@jilladams We've completed the work we need to do on our end, so confirming that we are good on our end for the increase to 100% on Monday 6/13 afternoon.

**Comment by jilladams** [comment]
Ok great! We had noted 6/13 or later, confirming that we'll get it done
next week and update ticket when it's complete.

**Comment by jilladams** [comment]
@allisonlu this ticket wasn't assigned during sprint, but Ian verified yesterday (in a comment I thought I posted but have now lost!) that they would appreciate if we can get this update live by Thurs EOD so his team has Friday to monitor. I know you picked up work on another ticket today -- any chance you'd have capacity to make this update on Wed for Thurs push?

**Comment by wesrowe** [comment]
@ianpsant, this PR is stuck along with all PRs in the vets-website repo. Platform teams are looking into the failure of an element of the release pipeline.

**Comment by jilladams** [comment]
@ryguyk THANK YOU! I thought i was going nuts.

**Comment by ianpsant** [comment]
@wesrowe Let me get back to you first thing tomorrow morning on whether we're ok with a Friday merge.

**Comment by jilladams** [comment]
vets-website merge/deploys are now unblocked: https://dsva.slack.com/archives/C01UB5A823V/p1655325553206689

**Comment by allisonlu** [comment]
Now on prod: https://www.va.gov/contact-us/
